env/env.cc

```diff
@@ -472,4 +480,14 @@ Status NewEnvLogger(const std::string& fname, Env* env,
   return Status::OK();
 }
 
+const std::shared_ptr<FileSystem>& Env::GetFileSystem() const {
+  return file_system_;
+}
+
+#ifdef OS_WIN
```

Comment:
I guess this function is here temporarily. It will eventually be moved to port/win?

Reply:
Yes. Eventually, WinEnv should be ported to the new APIs and this function will have a proper implementation.